### ISSUE TITLE
Inspect blockstack.js version in browser for referring app

### ISSUE THREAD

**Issue by stackatron** [blockstack/blockstack-browser#1797]
It is currently challenging to determine which apps have the latest blockstack.js. These bugs tend to cause onboarding errors. Ideas: 

• Blockstack.js change where we can easily expose/broadcast the version number in the browser, even if the app developer compiles in some funky way. 
• Have browser pick up that version number and display it someplace in the browser/auth flow. Assume it would be some kind of hidden or subtle way. 

This would allow end users and app reviewers to double check the version. Make testing easier. Maybe allow us to include version number in some kind of App Mining ranking. And also quickly determine the cause of bugs when they are related to version.

**Comment by markmhx** [comment]
Perhaps we can have the library output its version to the browser's console log during initiation so only those looking to debug the version will see it?

@hstove Have any thoughts on the best approach here? Any reason not to use the console in this manner in production for apps by default?

**Comment by hstove** [comment]
Yes, I think I'd prefer to have this be in the library itself, exposed as a global variable, like BLOCKSTACK_JS_VERSION.

The question is who the target user is. Is it us at PBC, trying to diagnose the error? If so, then we could just go to their app website and check that variable. Same with an app reviewer. I would question if an end user would ever want/need to see this.

If it was exposed as a global variable on their website, it would also make automated scraping of this data much easier, vs. in the auth flow.

**Comment by markmhx** [comment]
I'm moving this to the backlog since we had a clear need for this recently and will probably want it again soon.
